cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1255,7 +1255,17 @@
     </description>
   </property>
 
-  
+  <!-- Market Configuration -->
+
+  <property>
+    <name>market.base.url</name>
+    <value>http://market.cask.co/v2</value>
```

Comment:
@sreevatsanraman it will have to be v2 in 4.1, right? I thought that was the consensus. Otherwise there's no good way to add one click deploy currently?